microsoft-edge/webview2/how-to/debug-visual-studio.md

```diff
@@ -35,59 +35,96 @@ To debug your code, first install and set up the script debugger tools in Visual
 <!-- ====================================================================== -->
 ## Install the JavaScript diagnostics component
 
-First, install the **JavaScript diagnostics** component in **Desktop development with C++**, as follows.
+First, install the **JavaScript diagnostics** component in the **Desktop development with C++** workload of Visual Studio, as follows.
+
+
+<!-- ------------------------------ -->
+#### If you're installing Visual Studio now
+
+1. To work with the WebView2 samples, download Visual Studio 2019 (version 16.4 Preview 2 or later), from [Downloads](https://visualstudio.microsoft.com/downloads/), and then run the downloaded installer file.  To work with your own WebView2 app, you might want to do these steps for Visual Studio 2022 as well.
+
+1. In the Visual Studio Installer, select the **Desktop development with C++** workload, so that a checkmark appears, similar to the screenshot in the next section.
+
+1. In the tree of components for the **Desktop development with C++** workload, on the right, select the **JavaScript diagnostics** component, similar to that checkbox shown in the screenshot in the next section.
+
+1. Click the **Install** button.  When the installer finishes, you can reboot your machine to clean up, as the installer suggests.
+
+See also:
+* [Install Visual Studio](./machine-setup.md#install-visual-studio) in _Set up your Dev environment for WebView2_.
+
+Continue with [Enable script debugging for WebView2 apps](#enable-script-debugging-for-webview2-apps), below.
+
+
+<!-- ------------------------------ -->
+#### If you've already installed Visual Studio
+
+To work with most of the WebView2 samples, use Visual Studio 2019.  To work with your own WebView2 app, you might want to do these steps for Visual Studio 2022 as well.
```

Comment:
Same situation - Visual Studio 2022 will suffice for both sample apps and own WebView2 app